charts/generic-service/Chart.yaml

```diff
@@ -7,4 +7,4 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.6.0
+version: 2.0.0
```

Comment:
Could you update https://github.com/ministryofjustice/hmpps-helm-charts/releases to add some notes on v 2.0.0?